Dockerfile

```diff
@@ -18,9 +18,7 @@ COPY --from=builder /app .
 RUN pip install --upgrade pip && \
     pip install -r requirements.txt
 
-RUN python manage.py migrate
-
 EXPOSE 8080
 
 # Run database migrations and start the Django application
-ENTRYPOINT ["python", "manage.py", "runserver", "0.0.0.0:8080"]
+ENTRYPOINT ["sh", "c", "python manage.py migrate && python manage.py", "runserver", "0.0.0.0:8080"]
```

Comment:
Incorrect using option for sh. Fix to suggestion that you can watch in next line
`ENTRYPOINT ["sh", "-c", "python manage.py migrate && python manage.py", "runserver", "0.0.0.0:8080"]`

Reply:
@Alkaponees I've changed